test/conftest.py

```diff
@@ -1,263 +1,826 @@
 # pragma pylint: disable=missing-docstring
 import pytest
+import responses
 from autoprotocol.container import Container
 from autoprotocol.container_type import ContainerType
-from autoprotocol.unit import Unit
 from autoprotocol.protocol import Protocol
 
 
+@pytest.fixture(scope="module", autouse=True)
+def run_around_tests():
+    print("Define mock")
+
+    with responses.RequestsMock(assert_all_requests_are_fired=False) as rsps:
+        rsps.add(
+            rsps.GET,
+            "https://secure.strateos.com/api/container_types/res-sw96-hp",
```

Comment:
nit: this is a pretty huge function,  would prefer defining it in a separate file and importing it, or directly saving the response as a json file and deserialize it here